src/services/FileSystemService.ts

```diff
@@ -15,7 +15,7 @@ export class FileSystemService implements IProjectFilesBrowserService {
     }
   }
 
-  readDirectory(path: string) {
+  async readDirectory(path: string) {
```

Comment:
Maybe I remember it wrong, but I have a feeling that we've already discussed if it should be async or not and agreed, that the async shouldn't be there. Or? It shouldn't be working before adding, should it?

Reply:
Yes, we have to remove it. It's not necessary.

Reply:
Will you remove it or should I?

Reply:
Done